frame/contracts/src/migration/v12.rs

```diff
@@ -80,6 +80,7 @@ pub struct CodeInfo<T: Config> {
 	#[codec(compact)]
 	refcount: u64,
 	determinism: Determinism,
+	code_len: u32,
```

Comment:
This migration has landed in master but is not part of a release yet, so that's probably ok to add this quick addition to avoid another costly migration

Reply:
please make sure v12 isn't released before merging to master